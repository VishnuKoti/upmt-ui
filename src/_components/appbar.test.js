import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount } from 'enzyme';
import AppBar from './appbar';

describe('testing navigation', () => {
    it('appbar should render correctly', () => {
     const appbar = shallow(<AppBar />);
     appbar.render();
     expect(appbar).toMatchSnapshot();
    });
   });