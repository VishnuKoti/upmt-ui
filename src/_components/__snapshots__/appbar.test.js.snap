// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing navigation appbar should render correctly 1`] = `ShallowWrapper {}`;
